packages/manager/modules/dedicated-universe-components/src/directives/checkboxSwitch/index.js

```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+
+import checkboxSwitchDirective from './checkboxSwitch';
+import './checkboxSwitch.css';
+import './checkboxSwitch-2.css';
+
+const moduleName = 'ducCheckboxSwitch';
+
+angular
+  .module(moduleName, [])
+  .directive('checkboxSwitch', checkboxSwitchDirective);
```

Comment:
```bash
 $ git grep checkbox-swicth`
 packages/manager/apps/web/client/app/hosting/ftp/FTP.html:                                data-checkbox-switch3
````
so it looks we can safely remove this component from dedicated components

Reply:
removed